test/ponse.js

```diff
@@ -112,3 +112,28 @@ test('ponse: send', (t) => {
     t.end();
 });
 
+test('ponse: get: content-type', async (t) => {
+    const filesToMimes = {
+        "320px-Floppy_disk_2009_G1" : "image/jpeg",
+        "320px-Floppy_disk_2009_G1.jpg" : "image/jpeg",
+        "294px-Railroad1860.png" : "image/png",
+        "294px-Railroad1860" : "image/png",
+        "hello.txt" : "text/plain; charset=UTF-8",
+    }
+
+    var file;
+    for (file in filesToMimes) {
```

Comment:
```suggestion
    for (const file of keys(filesToMimes)) {
```

Reply:
Could you please squash to one commit :)?

Reply:
There will be a few commits in between to incorporate your changes, but once all is ready I will do a squash.